client/web/one/public/custom.css

```diff
@@ -0,0 +1,3 @@
+#one {
+ color: blueviolet;
```

Comment:
Remove this file and my commit please. 
Also if you can spend some time figuring out the sass implementation. Would be really good if we can have it in v1